contract/proxy-oracle/soroban/scripts/make_release_manifest.py

```
@@ -0,0 +1,264 @@
+    """Return the stellar contract install command (dry-run template, no broadcast)."""
+    return (
+        f"stellar contract install"
+        f" --wasm {wasm_path}"
+        f" --network {network}"
+        f" --source {source}"
+        f" --simulate-only"
+    )
+
+
+def build_initialize_command(
+    wasm_hash: str,
```

Ruff (0.15.15)

[warning] Unused function argument: `wasm_hash`

(ARG001)